build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.17.7
+FROM nginx:1.17.7 AS base
 
 # forward nginx access and error logs to stdout and stderr of the ingress
 # controller process
```

Comment:
I've never seen `/proc/1/fd/1` being used instead of `/dev/stdout` in a Dockerfile and looks weird to me 😄. Any chance you could change it since you're modifying the files anyway? Also, the nginx base image already has the forward for error and access log, so we really need just the one for `stream-access.log`.